File: 백준/Bronze/2292. 벌집/벌집.cs
using System;

class Program {
    static void Main(){
        int cnt = 1;
        int layer = 1;
        int n;
        n = int.Parse(System.Console.ReadLine());

        while(n > cnt){
            cnt += 6*layer;
            layer++;
        }
        System.Console.WriteLine(layer);
    }
}

File: 백준/Bronze/2292. 벌집/벌집.cc
#include <iostream>

int main()
{
    int cnt = 1;
    int layer = 1;
    int n;

    std::cin >> n;

    while(n > cnt){
        cnt += 6*layer;
        layer++;
    }
    std::cout << layer;
    
    return 0;
}